src/utils/error.js

```diff
@@ -0,0 +1,31 @@
+import logger from './winston';
+/**
+ * Class representing errorHandlers.
+ * @extends Error
+ */
+class ErrorHandler extends Error {
+  /**
+     * Create a dot.
+     * @param {number} statusCode - The status code of the error.
+     * @param {string} message - The message to be passed for the error.
+     */
+  constructor(statusCode, message) {
+    super();
+    this.statusCode = statusCode;
+    this.message = message;
+  }
+}
+
+const handleError = (err, res) => {
```

Comment:
This looks good, but I think this method tightly coupled to express's `Response` object ie. (it depends on `res` to work). How would you typically handle errors that are outside the context of `express`?

Reply:
About the comment so I should remove the status and the res  so as to handle all errors generally??

Reply:
@timolinn I implemented that it throws a 500 error and logs the error stack to the logger file is that okay??